crates/re_renderer/src/resource_managers/resource_manager.rs

```diff
@@ -42,6 +42,9 @@ pub enum ResourceManagerError {
 
     #[error("Failed accessing resource pools")]
     ResourcePoolError(PoolError),
+
+    #[error("Invalid mesh given as input")]
+    InvalidMesh(#[from] crate::mesh::MeshError),
```

Comment:
The other errors are all about resource managers in general, so this _should_ go to the mesh manager specifically.
However, the abstracted resource manager idea broke down quite a bit - e.g. I think I'll soon remove the "temp vs permanent resource" distinction; this didn't play out as I originally envisioned. So I'm ok if we punt on this :)